numpy/lib/arrayterator.py

```diff
@@ -28,8 +28,8 @@ class Arrayterator:
 
     Parameters
     ----------
-    var : array_like
-        The object to iterate over.
+    var : array
+        The array to iterate over.
```

Comment:
This change looks incorrect. From a few lines above: _"`Arrayterator` can be used with any object that supports multidimensional slices."_

Reply:
That's a good point actually, I'll revert the change.